doc/src/user_guide.rst

```diff
@@ -1528,6 +1528,65 @@ Example::
     int-E = rho*(u*u + v*v + w*w)
     int-enst = rho*(%(vor1)s*%(vor1)s + %(vor2)s*%(vor2)s + %(vor3)s*%(vor3)s)
 
+[soln-plugin-ascent]
+^^^^^^^^^^^^^^^^^^^^
+
+Uses `Alpine Ascent <https://github.com/Alpine-DAV/ascent>`_ to plot 
+on-the-fly. You will need both Conduit and Ascent compiled with MPI 
+and VTK-h support. The paths can then be set in the terminal with::
+
+    export PYFR_CONDUIT_LIBRARY_PATH=/path/to/libconduit.so
+    export PYFR_ASCENT_MPI_LIBRARY_PATH=/path/to/libascent_mpi.so
+
+The following parameters can then be set:
+
+1. ``nsteps`` --- produce the plots every ``nsteps`` time steps:
+
+    *int*
+
+2. ``division`` --- the level of linear subdivison to use
+
+    *int*
+
+3. ``scene-{name}`` --- a scene to plot with ascent options passed in a 
+dictionary. The field value can be a valid expression and rendering options can
+be set as ``render-ascent/option`` where ``ascent/option`` is some valid
+rendering option. Within the dictionary a field expression can be set and named,
+or a field can be named if it has already been defined in another scene defined. 
+
+    *dict*
+
+4. ``pipeline-{name}`` --- a pipeline of data manipulations that can be used 
+within a scene. The value is a dictionary containing the valid configuration 
+options. Pipelines that require implict fields are supported and will
+automatically add fields to the underlying data structure is they are not 
+already defined. For example, the Q-criterion pipeline type implicitly 
+requires a vector field called ``velocity``. 
+
+   *dict*
+
+5. ``field-{name}`` --- this is an extension to the Ascent library, where we
+allow users to manually add fields to the underlying datastructure. This is 
+sometimes useful when working with pipelines that output data. The input is a
+dict which has keys ``field`` and ``field_expr``/ ``field_expr_[x,y,(z)]``, 
+with the later used to define vector expressions. 
+
+    *dict*
+
+Example::
+
+    [soln-plugin-ascent]
+    nsteps = 200
+    scene-ke = {'image_name': 'ke-{t:.1f}', 'field_expr': '0.5*rho*(u*u + v*v)', 'field': 'ke', 'type': 'pseudocolor'}
```

Comment:
Consistency between _ and -.

Reply:
The issue is that ascent uses _ and pyfr uses -, so to make users life easier when reading the ascent docs I went with _ in the dict but stuck to - in the rest of the ini.